### PR TITLE
fix(kirby): show web3enable view on saved preference

### PR DESCRIPTION
**Pull request by dankins**
if logged out of portis from another window and then reload the page the portis login screen will be displayed from the iframe, but the kirby view is not displayed so the user cannot do anything.

### CODE REVIEW

File: packages/kirby/src/views/Web3Enable.tsx [hunk 1]
```diff
@@ -8,32 +11,60 @@ import {
...
+      (ctx.core.plugins.view as ViewPlugin).completeView();
+    }
   }
+
+  if (status === "provided") {
+    return <></>;
+  } else if (status === "enabling provider") {
+    return (
+      <CenteredPage>
+        <ClipLoader />
```
Comment (nickreynolds) [inline packages/kirby/src/views/Web3Enable.tsx]: can we add some text here like how we have "waiting for wallet connection" and "waiting for signuatre"?

Comment (dankins) [reply packages/kirby/src/views/Web3Enable.tsx]: good call, added

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.29 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**